src/views/LoginView.vue

```diff
@@ -1,35 +1,94 @@
 <template>
   <div>
     <main>
-      <img id="login-image" src="../assets/img/bier.png" alt="logo" />
+      <img
+          id="login-image"
+          src="../assets/img/bier.png"
+          alt="logo"
+      />
       <h1>{{ $t('login.SudoSOS Login') }}</h1>
-      <Button id="login-gewis-button" @click="loginViaGEWIS" severity="success"
-        ><img id="gewis-branding" src="../assets/img/gewis-branding.svg" alt="GEWIS" />{{
-          $t('login.Login via GEWIS')
-        }}</Button
+      <Button
+          id="login-gewis-button"
+          @click="loginViaGEWIS"
+          severity="success"
       >
-      <hr />
-      <form id="login-form" @submit.prevent="ldapLogin">
-        <!--      TODO: Form validation with vee-validate -->
-        <label for="username">{{ $t('login.Username') }}</label>
-        <InputText
-          id="username"
-          type="text"
-          v-bind="username"
-          :placeholder="$t('login.Enter username')"
-        />
-        <label for="password">{{ $t('login.Password') }}</label>
-        <InputText
-          id="password"
-          type="password"
-          v-bind="password"
-          :placeholder="$t('login.Enter password')"
+        <img
+            id="gewis-branding"
+            src="../assets/img/gewis-branding.svg"
+            alt="GEWIS"
         />
-        <Button type="submit" id="login-button" severity="danger">{{ $t('login.Login') }}</Button>
-        <div class="password-reset" @click="resetPassword">{{ $t('login.Password reset') }}</div>
-      </form>
+        {{ $t('login.Login via GEWIS') }}
+      </Button>
+      <hr />
+      <Form id="login-form" @submit="ldapLogin">
+        <label
+            id="input-description"
+            for="username"
+        >
+          {{ $t('login.Username') }}
+        </label>
+        <span class="p-float-label with-error">
```

Comment:
Voelt alsof we hier een component van kunnen maken? er zit best wat overlap tussen die van de username als die van t password; alleen misschien is een component weer kut met binding enzo. food for though.

Reply:
Zou kunnen, maar lijkt me iets voor later, aangezien het me een typisch iets lijkt wat dagen gaat kosten omdat je net niet helemaal weet hoe het in elkaar steekt. Zou hiervoor wel een issue openen.